src/System.Numerics.Vectors/src/System/Numerics/Quaternion.cs

```diff
@@ -121,14 +95,9 @@ public static Quaternion Normalize(Quaternion value)
         /// <returns>A new Quaternion that is the conjugate of the specified one.</returns>
         public static Quaternion Conjugate(Quaternion value)
         {
-            Quaternion ans;
-
-            ans.X = -value.X;
-            ans.Y = -value.Y;
-            ans.Z = -value.Z;
-            ans.W = value.W;
-
-            return ans;
+            Vector4 q = -ToVector4(value);
```

Comment:
I did a quick Benchmark on this change:

```C#
        [Benchmark]
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        public static void Conjugate()
        {
            Quaternion start = new Quaternion(8.5f, 9.4f, 1.2f, 1f);

            Quaternion c1 = Quaternion.Conjugate(start);
            Quaternion c2 = Quaternion.Conjugate(c1);
            Quaternion c3 = Quaternion.Conjugate(c2);
            Quaternion c4 = Quaternion.Conjugate(c3);
        }
```

And the results show a degradation of this method on my machine:

```
BenchmarkDotNet=v0.10.10.20171127-develop, OS=Windows 10 Redstone 3 [1709, Fall Creators Update] (10.0.16299.19)
Processor=Intel Core i7-6700 CPU 3.40GHz (Skylake), ProcessorCount=8
Frequency=3328122 Hz, Resolution=300.4698 ns, Timer=TSC
.NET Core SDK=2.2.0-preview1-007522
  [Host]     : .NET Core 2.1.0-preview1-25907-02 (Framework 4.6.25901.06), 64bit RyuJIT
```

With your changes:

Method |     Mean |     Error |    StdDev |
---------- |---------:|----------:|----------:|
 Conjugate | 57.38 ns | 0.9437 ns | 0.8827 ns |

Without the changes:

Method |     Mean |     Error |    StdDev |
---------- |---------:|----------:|----------:|
 Conjugate | 5.740 ns | 0.0549 ns | 0.0397 ns |

Reply:
I also see a degradation for Normalize. Same machine as above.

```C#

        [Benchmark]
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        public static void Normalize()
        {
            Quaternion start = new Quaternion(8.5f, 9.4f, 1.2f, 1f);

            Quaternion c1 = Quaternion.Normalize(start);
            Quaternion c2 = Quaternion.Normalize(c1);
            Quaternion c3 = Quaternion.Normalize(c2);
            Quaternion c4 = Quaternion.Normalize(c3);
        }
```

With your Normalize change:

Method |       Mean |     Error |    StdDev |
---------- |-----------:|----------:|----------:|
 Normalize | 109.890 ns | 1.6731 ns | 1.5650 ns |

Without your Normalize change:

Method |      Mean |     Error |    StdDev |
---------- |----------:|----------:|----------:|
 Normalize | 60.797 ns | 0.3803 ns | 0.3557 ns |

Reply:
It was better with the Unsafe casting; but produced the wrong results in release 😢 
```
           Method |      Mean |     Error |    StdDev | Scaled | ScaledSD |
----------------- |----------:|----------:|----------:|-------:|---------:|
  ConjugateUnsafe |  8.564 ns | 0.0385 ns | 0.0322 ns |   0.31 |     0.00 |
 ConjugateCurrent | 27.613 ns | 0.1639 ns | 0.1533 ns |   1.00 |     0.00 |
  ConjugateChange | 64.814 ns | 0.2741 ns | 0.2564 ns |   2.35 |     0.02 |
```
Will have to dig into why its producing wrong results.

Reply:
Are there issues if you make this a union, rather than using `Unsafe.As`?

Reply:
I'm fairly sure (can't find code atm) that the Jit won't consider struct with overlapping fields for a register

Reply:
Will try to get a repo and file in coreclr; then revert this back to the unsafe version

Reply:
Raised issue https://github.com/dotnet/coreclr/issues/15237